privacyidea/lib/tokens/emailtoken.py

```diff
@@ -304,9 +304,9 @@ def create_challenge(self, transactionid=None, options=None):
                     mimetype=mimetype)
 
             except Exception as e:
-                info = ("The PIN was correct, but the "
-                        "EMail could not be sent: %r" % e)
-                log.warning(info)
+                info = _("The PIN was correct, but the "
+                         "EMail could not be sent")
+                log.warning(info + "({0!s})".format(e))
```

Comment:
We should have consistent strings. Here is no punctuation, in the SMSToken there is a full-stop. Maybe an exclamation mark would be fitting here.
Also there is no space between the info string and the added error message in the warning.